docs/windows/includes/target-windows.md

```diff
@@ -0,0 +1,10 @@
+---
+ms.topic: include
+ms.date: 11/07/2024
+---
+
+## Target Windows
+
+In Visual Studio, set the **Debug Target** to **Framework (...)** > **net8.0-windows**. There is a version number in the item entry, which may or may not match the following screenshot:
```

Comment:
This should say net9.0-windows for .NET 9 version of the doc.

Reply:
Yep, I've got a separate task to update net8-*, but I might as well fix it here now.